src/megaapi_impl.cpp

```diff
@@ -16262,6 +16454,16 @@ long long MegaAccountDetailsPrivate::getNumFolders(MegaHandle handle)
     return details.storage[handle].folders;
 }
 
+long long MegaAccountDetailsPrivate::getVersionStorageUsed(MegaHandle handle)
+{
+    return details.storage[handle].version_bytes;
```

Comment:
The struct should be automatically initialized to zero if it's created that way. Anyway I agree that it would be better to check if the handle exist, using `find` to get the iterator and using it (to not look for the handle twice). Not critical IMO, but it would be good for sure.

Reply:
I've added a commit to do it as suggested above ;)